saleor/extensions/plugins/avatax/plugin.py

```diff
@@ -182,30 +189,31 @@ def _calculate_checkout_subtotal(
     def calculate_checkout_subtotal(
         self,
         checkout: "Checkout",
-        discounts: List["DiscountInfo"],
+        discounts: "DiscountsListType",
         previous_value: TaxedMoney,
     ) -> TaxedMoney:
         self._initialize_plugin_configuration()
 
         if self._skip_plugin(previous_value):
             return previous_value
 
-        sub_total = checkout.get_subtotal(discounts)
+        # TODO: Isn't previous value for that?
```

Comment:
@korycins wdyt?

Reply:
I think that we should use `previous_value` here. But it doesn't solve our problem. Avatax calculates prices on its own. It doesn't take into account values from the previous plugins. It will be hard to change as Avatax requires a lot of information about single products...